src/ui/order/Basket.tsx

```diff
@@ -0,0 +1,68 @@
+import { Loading, useGetMany, useRecordContext } from "react-admin";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import Paper from "@mui/material/Paper";
+import Link from "@mui/material/Link";
+
+export const Basket = () => {
+  const order = useRecordContext();
+  if (!order) return null;
+
+  const productIds = order.basket.map((b) => b.product_id);
+  const { data: products, isLoading } = useGetMany("products", {
+    ids: productIds,
+  });
+
+  if (isLoading) return <Loading />;
+
+  if (!products) return null;
+
+  const representation = products.map((product) =>
+    Object.assign(
+      product,
+      order.basket.find((b) => b.product_id === product.id)
+    )
+  );
```

Comment:
I don't understand this...

Reply:
This is for getting data of products and order; together

This result in that : 
```json
[
  {
    "id": 108,
    "category_id": 10,
    "reference": "Eye Rest",
    "width": 26.15,
    "height": 30.87,
    "price": 41.68,
    "thumbnail": "https://marmelab.com/posters/tech-9.jpeg",
    "image": "https://marmelab.com/posters/tech-9.jpeg",
    "description": "Temporibus consequatur sunt praesentium qui. Amet doloremque illo. Minima nobis suscipit voluptas.",
    "stock": 0,
    "sales": 18,
    "product_id": 108,
    "quantity": 1
  }
]
```
Maybe there is an easier way ?

Reply:
In any case, assign to an empty object at least to avoid mutating `product`